cypress/integration/team-members/list.js

```diff
@@ -0,0 +1,33 @@
+describe('List team members', () => {
+  before(() => {
+      cy.task('db:seedTeamMembers')
+    }
+  )
+  it('Should not have permissions when not logged in', () => {
+    cy.visit('http://localhost:3000/team-members/list')
+
+    cy.contains('Welcome to SalesWell')
+  })
+  
+  it('Should have permissions when logged in as user', () => {
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/team-members/list')
+
+    cy.contains('admin user').should('exist')
+    cy.contains('admin@example.com').should('exist')
+    cy.contains('ADMIN').should('exist')
+
+    for (let i = 10; i >= 1; i--) {
```

Comment:
Let's make two `for` loops, one for normal users and one for admin users, is more clear. Also, let's use some comments.